courses/B.yaml

```diff
@@ -46,6 +46,7 @@
   reading: |
     >Introduction to `LaTeX <https://tobi.oetiker.ch/lshort/lshort.pdf>`_ (online book)
     >Short summary of `LaTex features <https://www.latex-tutorial.com/quick-start/>`__
+    >`Guide to LaTeX <https://www.cs.helsinki.fi/u/jhasa/latexkurssi/>`__ (FINNISH)
```

Comment:
I'd suggest we move this to the Aalto column... while it's more broad than that, it's a useful place while we figure out what to do with other languages.

I also always do `(FI)` instead of the whole name... might save some space...

Reply:
acknowledged